tests/wpt/meta/css/css-position/sticky/position-sticky-table-td-left.html.ini

```diff
@@ -1,2 +1,2 @@
 [position-sticky-table-td-left.html]
-  expected: FAIL
+  expected: TIMEOUT
```

Comment:
Are these tests that have TIMEOUT  just crashes that don't exit servo?  Or do they indicate that implementation is slow on CI?

Reply:
Nope. I think these are failing for some other reason. I do see this error when running them:

```
 [2023-11-28T17:07:01Z ERROR script::dom::bindings::error] Error at http://web-platform.test:8000/css/css-position/resources/ref-rectangle.js:14:3 document.fonts is undefined
```

So I suspect it's related to the lack of `documents.fonts` support.